helm/scylla-manager/templates/controller_deployment.yaml

```diff
@@ -36,7 +36,7 @@ spec:
               fieldPath: metadata.namespace
         args:
         - manager-controller
-        - --log-level={{ .Values.logLevel }}
+        - --loglevel=2
```

Comment:
now I recall that was intentional because the value was pasted into both manager and manager controller, yet those are distinct apps, I choose to retain it for the operand

Reply:
also the comment said it's for the operand `# Scylla Manager log level`